content/integrations/bugsnag.md

```diff
@@ -23,15 +23,23 @@ No installation is required.
 
 To integrate Bugsnag with Datadog:
 
-1. In [Bugsnag](https://bugsnag.com/) go to _Settings_ for the project you would like to configure to send notifications to Datadog
-1. Select _Team Notifications_ and then _Datadog_
-1. Customize the notifications you'll see in Datadog by selecting error notification triggers.
-1. Apply custom filters to your notification triggers to see errors from specific release stages and severities.
-1. Enter your Datadog API key.
-1. Select _Test Notification_ to test the configuration. A test error from Bugsnag will appear in Datadog's event stream.
-1. _Save_ your settings.
-1. Add more streams from the same project to see error events based on a different set of notification criteria.
+1. Go to `Settings` in [Bugsnag](https://bugsnag.com/)  for the project you would like to configure to send notifications to Datadog.
 
+2. Select `Team Notifications` and then `Datadog`.
```

Comment:
Again, **Team Notifications** and **Datadog**, not `Team Notifications` and `Datadog`